src/Numerics/Mesh/Interpolation.jl

```diff
@@ -225,9 +224,9 @@ struct InterpolationBrick{
 
         end # el loop
 
-        m_ξ1, _ = GaussQuadrature.legendre(FT, qm[1], GaussQuadrature.both)
-        m_ξ2, _ = GaussQuadrature.legendre(FT, qm[2], GaussQuadrature.both)
-        m_ξ3, _ = GaussQuadrature.legendre(FT, qm[3], GaussQuadrature.both)
+        m_ξ1, _ = Elements.lglpoints(FT, qm[1]-1)
+        m_ξ2, _ = Elements.lglpoints(FT, qm[2]-1)
+        m_ξ3, _ = Elements.lglpoints(FT, qm[3]-1)
```

Comment:
You could special case this and for `N=0`  use `glpoints`.

Reply:
Done.